src/styles/blocks/card.scss

```diff
@@ -0,0 +1,98 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &__img {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__description {
+    margin: 0 0 4px;
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    color: $color-accent-dark;
+  }
+
+  &__code {
+    color: $color-secondary-gray;
+    margin-bottom: 16px;
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: 400;
+  }
+
+  &_review {
+    box-sizing: border-box;
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    margin-bottom: 24px;
+
+    &-value {
+      font-weight: 400;
+      font-size: 10px;
+      color: $color-accent-dark;
+      line-height: 14px;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+
+    &-text {
+      font-size: 12px;
+      line-height: 18px;
+      color: $color-secondary-gray;
+    }
+
+    &-actual-price {
+      font-size: 16px;
+      line-height: 18px;
+      font-weight: bold;
+      color: $color-accent-dark;
+    }
```

Comment:
Wrong `&` usage, use it to avoid duplicated BEM block or element class names part, not just for combining two words